Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -126,6 +129,20 @@ public struct FeatureFormView: View {
                 title = newTitle
             }
         }
+        // The sheet view modifier needs to be used here or it will
+        // be dismissed on the first open by SwiftUI.
+        .sheet(isPresented: $model.fullScreenTextInputIsPresented) {
+            if let element = model.focusedElement as? FieldFormElement {
+                FullScreenTextInput(
+                    text: $model.fullScreenText,
+                    element: element,
+                    model: model
+                )
+#if targetEnvironment(macCatalyst)
+                .environmentObject(model)
+#endif
+            }
+        }
```

Comment:
This is unfortunate. Maybe it would be worth putting this into an internal modifier to keep `evaluatedForm` as slim as possible by reducing this to 1 line.

Reply:
Yeah it is a bit odd that this sheet modifier had to be on the `Form` to work properly while other sheets in feature form work properly. 

> Maybe it would be worth putting this into an internal modifier to keep evaluatedForm as slim as possible by reducing this to 1 line.

I'm not sure I see much benefit of that right now but if I can make the change if that is what you want.

Reply:
I'm not sure if a sheet modifier is something that should be hidden in a custom view modifier. It seems like an important detail. Thoughts?